backend/src/data/repositories/worker/worker.repository.ts

```diff
@@ -0,0 +1,61 @@
+import { Worker as WorkerM } from '~/data/models/models';
+import { Worker as WorkerEntity } from '~/services/worker/worker.entity';
+
+type Constructor = {
+  WorkerModel: typeof WorkerM;
+};
+
+class Worker {
+  #WorkerModel: typeof WorkerM;
+
+  constructor({ WorkerModel }: Constructor) {
+    this.#WorkerModel = WorkerModel;
+  }
+
+  public async getAll(): Promise<Array<WorkerEntity>> {
+    const workers: WorkerM[] = await this.#WorkerModel.query().select('*');
+    return workers.map((worker) => Worker.modelToEntity(worker));
+  }
```

Comment:
You should remove all getAll functionality because it is already done in https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/pull/37